documentation/scope-commands.md

```diff
@@ -49,17 +49,17 @@ _More Info -_ On real ATC systems, moving the data block is sometimes used by co
 
 ~~_More Info -_ **THIS COMMAND IS NOT YET AVAILABLE**~~
 
-~~### Set Scratch Pad~~
-~~_Syntax -_ `[scratchpad text] [CID]`~~
+### Set Scratch Pad~~
+_Syntax -_ `[scratchpad text] [CID]`
 
-~~_Description -_ This will amend the value stored in the target's data block's scratch pad. This value can be a maximum of three alphanumeric characters.~~
+_Description -_ This will amend the value stored in the target's data block's scratch pad. This value can be a maximum of three alphanumeric characters.~~
 
-~~_More Info -_ **THIS COMMAND IS NOT YET AVAILABLE**~~
+_More Info -_
 
-~~### Toggle Halo~~
-~~_Syntax -_ `QP_J [CID]`~~
-~~_Shortcut -_ `[F7] [CID]`~~
+### Toggle Halo
+_Syntax -_ `QP_J [CID]` or `QP_J [radius] [CID]`
+_Shortcut -_ `[F7] [CID]` or `[F7] [radius] [CID]`
 
-~~_Description -_ This toggles a circle around the center of the radar target, with a radius definable in the scope's theme (generally 3nm for terminal, or 5nm for en route).~~
+_Description -_ This toggles a circle around the center of the radar target, with a radius you can specify in the command. If the radius is not specified, the default halo size will be used (generally 3nm for terminal, or 5nm for en route).
 
-~~_More Info -_ **THIS COMMAND IS NOT YET AVAILABLE**~~
+_More Info -_ Note that if a resized halo exists, entering `[F7] [CID]` will actually be interpreted by the system as `[F7] 3 [CID]`-- meaning the halo will be resized to 3nm, not removed. With the halo back to 3nm, entering `[F7] [CID]` again will remove the halo. If you add a 10nm halo with `[F7] 10 [CID]`, it can be removed instead of resized by specifying `[F7] 10 [CID]`, or use the above method of entering `[F7] [CID]` twice.
```

Comment:
why 10nm?

Reply:
It's the standard behavior, per the vERAM user guide. It's not inherently 10, but rather that an attempt to draw a halo when one of that size currently exists will result in it being removed.
```
QP_J THY7 (draws 3nm ring)
QP_J 6 THY7 (adjusts ring to 6nm)
QP_J THY7 (adjusts to 3nm ring)
QP_J THY7 (removes ring)

QP_J 10 THY8 (draws 10 mile ring)
QP_J 10 THY8 (removes ring)
```

Reply:
My bad. I think I misread the text. Works for me